backend/kafka_backend/kafka_backend.go

```diff
@@ -0,0 +1,153 @@
+package kafkabackend
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/skroutz/downloader/job"
+)
+
+// FlushTimeout is the timeout we give to out kafka producer
+// to flush pending messages.
+const FlushTimeout = 5000
+
+var eventsWG sync.WaitGroup
```

Comment:
Having for example 2 kafka backends with different confis was not a prerequisite for this implementation. This will take time if we want to implement it.